src/legacy/core_plugins/telemetry/server/collection_manager.ts

```diff
@@ -141,7 +169,17 @@ export class TelemetryCollectionManager {
       return;
     }
 
-    return await collection.statsGetter(clustersDetails, statsCollectionConfig);
+    const stats = await collection.statsGetter(clustersDetails, statsCollectionConfig);
+    const licenses = await collection.licenseGetter(clustersDetails, statsCollectionConfig);
```

Comment:
can we group them in 1 `promise.all` so we dont have to wait for them sequentially?

Reply:
Good call! Just pushed the update :)